<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2024, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorMidiSysexForm</class>
 <widget class="QDialog" name="qtractorMidiSysexForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MIDI SysEx</string>
  </property>
  <property name="windowIcon" >
   <iconset theme="qtractorTracks" />
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" >
     <item rowspan="5" row="0" column="0" colspan="4" >
      <widget class="QTreeWidget" name="SysexListView" >
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights" >
        <bool>true</bool>
       </property>
       <property name="itemsExpandable" >
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Size</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Data (hex)</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QPushButton" name="ImportButton" >
       <property name="toolTip" >
        <string>Import from SysEx file</string>
       </property>
       <property name="text" >
        <string>&amp;Import...</string>
       </property>
       <property name="icon" >
        <iconset theme="formOpen" />
       </property>
      </widget>
     </item>
     <item row="1" column="4" >
      <widget class="QPushButton" name="ExportButton" >
       <property name="toolTip" >
        <string>Export to SysEx file</string>
       </property>
       <property name="text" >
        <string>E&amp;xport...</string>
       </property>
       <property name="icon" >
        <iconset theme="formSave" />
       </property>
      </widget>
     </item>
     <item row="2" column="4" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>4</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="4" >
      <widget class="QPushButton" name="MoveUpButton" >
       <property name="toolTip" >
        <string>Move SysEx item up on list order</string>
       </property>
       <property name="text" >
        <string>&amp;Up</string>
       </property>
       <property name="icon" >
        <iconset theme="formMoveUp" />
       </property>
      </widget>
     </item>
     <item row="4" column="4" >
      <widget class="QPushButton" name="MoveDownButton" >
       <property name="toolTip" >
        <string>Move SysEx item down on list order</string>
       </property>
       <property name="text" >
        <string>&amp;Down</string>
       </property>
       <property name="icon" >
        <iconset theme="formMoveDown" />
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QToolButton" name="OpenButton" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Open SysEx</string>
       </property>
       <property name="icon" >
        <iconset theme="formOpen" />
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QComboBox" name="NameComboBox" >
       <property name="minimumSize" >
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Sysex name</string>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2" >
      <widget class="QToolButton" name="SaveButton" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Save SysEx</string>
       </property>
       <property name="icon" >
        <iconset theme="formSave" />
       </property>
      </widget>
     </item>
     <item row="5" column="3" >
      <widget class="QToolButton" name="DeleteButton" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Delete SysEx</string>
       </property>
       <property name="icon" >
        <iconset theme="formRemove" />
       </property>
      </widget>
     </item>
     <item row="5" column="4" >
      <widget class="QPushButton" name="AddButton" >
       <property name="toolTip" >
        <string>Create SysEx item</string>
       </property>
       <property name="text" >
        <string>&amp;Add</string>
       </property>
       <property name="icon" >
        <iconset theme="formAdd" />
       </property>
      </widget>
     </item>
     <item rowspan="3" row="6" column="0" colspan="4" >
      <widget class="QTextEdit" name="SysexTextEdit" >
       <property name="font" >
        <font>
         <family>Monospace</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="4" >
      <widget class="QPushButton" name="UpdateButton" >
       <property name="toolTip" >
        <string>Update SysEx item</string>
       </property>
       <property name="text" >
        <string>Upda&amp;te</string>
       </property>
       <property name="icon" >
        <iconset theme="formEdit" />
       </property>
      </widget>
     </item>
     <item row="7" column="4" >
      <widget class="QPushButton" name="RemoveButton" >
       <property name="toolTip" >
        <string>Remove SysEx item</string>
       </property>
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
       <property name="icon" >
        <iconset theme="formRemove" />
       </property>
      </widget>
     </item>
     <item row="8" column="4" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>4</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Reset|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8" />
 <tabstops>
  <tabstop>SysexListView</tabstop>
  <tabstop>ImportButton</tabstop>
  <tabstop>ExportButton</tabstop>
  <tabstop>MoveUpButton</tabstop>
  <tabstop>MoveDownButton</tabstop>
  <tabstop>OpenButton</tabstop>
  <tabstop>NameComboBox</tabstop>
  <tabstop>SaveButton</tabstop>
  <tabstop>DeleteButton</tabstop>
  <tabstop>SysexTextEdit</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>UpdateButton</tabstop>
  <tabstop>RemoveButton</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtractor.qrc" />
 </resources>
 <connections/>
</ui>
